projects/vdk-plugins/vdk-control-api-auth/src/vdk/plugin/control_api_auth/auth_request_values.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from enum import Enum
+
+
+class AuthRequestValues(Enum):
```

Comment:
maybe mark with #deprecated the duplicates (e.g. `auth_request_values.py` in vdk-control-cli's auth package), so it's easier to diff current code with prior, and to locate what needs to be cleaned up afterwards?

Reply:
Will do as part of a separate PR.